content/components/action-list.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Action list
+status: Alpha
+---
+
+import {Grid, Flex, Box, Button, ButtonOutline, Heading} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    An action list is a vertical list of interactive actions or options. It’s composed of items presented in a consistent single-column format, with room for icons, descriptions, side information, and other rich visuals.
+</Box>
+
+<img src="https://user-images.githubusercontent.com/293280/122133975-96c1e000-cdf2-11eb-9ce9-02d5014c1831.png" alt="Action list examples" />
+
+<!--
+<ButtonOutline mr={1}>React component</ButtonOutline>
+<ButtonOutline mr={1}>ViewComponent</ButtonOutline>
+<ButtonOutline>Figma component</ButtonOutline>
+-->
+
+## Overview
+
+Action lists can have many applications. They’re the foundation of menus, selection panels, and other overlay components; they can be applied to page sidebars for showing individual actions, for handling local navigation, and for displaying metadata.
+
+Action lists support section dividers and headers for grouping items, and indivudal item dividers for added clarity.
+
+Action lists use a mobile-friendly inset style. Their sizes are adapted on touch devices, and their single-column format should render consistently in any screen size.
+
+Except for read-only and disabled exceptions, action list items are interactive.
+
+
+<!--
+<img alt="Action list wrapper structure" src="https://user-images.githubusercontent.com/223966/118818389-2e8fe500-b8ac-11eb-81b2-8a61f4b3d7ba.png" />
+<Caption>Action list wrapper structure</Caption>
+
+- **Action list item:** An action list item is a generic interactive element, which includes a main text and optional icons, descriptions, auxiliary information, and more.
+- **Action list section:** An action list section uses the ARIA role property to group elements from a list. Depending on the provided values, it renders a divider, a header, or both.
+- **Action list divider:** …
+- **Action list section header:** …
+
+<img alt="Action list item structure" src="https://user-images.githubusercontent.com/223966/118818435-3bacd400-b8ac-11eb-8691-ab9cbba73249.png" />
+<Caption>Action list item structure</Caption>
+
+
+
+## Application
+
+### In overlays
+
+### In pages
+
+
+-->
+
+## Anatomy 
+
+<img src="https://user-images.githubusercontent.com/293280/122128726-d7692b80-cde9-11eb-8e5b-2f4e2eb4b4a1.png" alt="Action list anatomy" />
```

Comment:
I think we need a small caption here, even if just:

An action list is composed of:
- Action list item
- Item dividers
- Section divider (subtle or filled)
- Section header (subtle or filled)